nslord/conf/pdns.conf.var

```diff
@@ -3,8 +3,8 @@ api=yes
 api-key=${DESECSTACK_NSLORD_APIKEY}
 default-soa-edit=INCREMENT-WEEKS
 default-soa-mail=hostmaster.desec.io
-default-soa-name=ns1.desec.io
-default-ttl=60
+default-soa-name=nslord
+default-ttl=3600
```

Comment:
If you mean "Django migration", we can't do that because our application is unaware of the existence of SOA records (and RRSIG, NSEC etc.). The existence of the legacy values is not really an issue though; I feel more like the new values are more future-proof. So I wouldn't bother writing a migration for now.

`nslord` is less specific in the sense that the value doesn't need to be changed when someone else deploys our stack. It's more like a class attribute now and not an instance attribute (if desec-stack is the class).

Reply:
alrighty